### PR TITLE
Improve Variation style ordering

### PR DESCRIPTION
**Pull request by pcln-james**

### CODE REVIEW

File: packages/core/src/Button/Button.js [hunk 1]
```diff
@@ -1,41 +1,28 @@
...
+    padding: 7px 12px;
+  `,
+  medium: css`
+    font-size: ${themeGet('fontSizes.1')}px;
+    padding: 9.5px 18px;
+  `,
+  large: css`
+    font-size: ${themeGet('fontSizes.2')}px;
+    padding: 12px 22px;
+  `,
```
Comment (pcln-james) [inline packages/core/src/Button/Button.js:25]: gave Button more of an overhaul that the others, it now supports responsive sizing 😄 
`size={['small', 'medium', 'large']}`

File: packages/core/src/CloseButton/__snapshots__/CloseButton.spec.js.snap [hunk 2]
```diff
@@ -27,8 +27,6 @@ exports[`CloseButton renders without props 1`] = `
   border-radius: 2px;
   border-width: 0;
   border-style: solid;
-  font-size: 14px;
-  padding: 9.5px 18px;
```
Comment (craigpalermo) [inline packages/core/src/CloseButton/__snapshots__/CloseButton.spec.js.snap:31]: Is this snapshot update expected?

Comment (pcln-james) [reply packages/core/src/CloseButton/__snapshots__/CloseButton.spec.js.snap:31]: I'm not sure why this got removed but these styles have zero impact on this component, the padding is overwritten below and the font-size doesn't affect the icon.

**Review by craigpalermo — APPROVED** [review]

File: packages/modal/src/Headers.spec.js [hunk 3]
```diff
@@ -1,35 +1,17 @@
 import React from 'react'
-import { render } from '@testing-library/react'
+import { render } from 'testing-library'
```
Comment (sdalonzo) [inline packages/modal/src/Headers.spec.js:2]: 🙏

**Review by sdalonzo — APPROVED** [review]
thanks for this! LGTM

**Pull request merged**